frontend/ui/elements/page_turns.lua

```diff
@@ -122,6 +122,18 @@ local PageTurns = {
                     end,
                 })
             end,
+        },
+        {
+            text = _("Also invert document-related dialogs"),
+            checked_func = function()
+                return G_reader_settings:isTrue("invert_ui_layout_mirroring")
+            end,
+            callback = function()
+                G_reader_settings:flipNilOrFalse("invert_ui_layout_mirroring")
+            end,
+            help_text = _([[
+When enabled the UI direction for the Table of Contents, Book Map, Page Browser, and Skim dialogs will follow the page turn direction instead of the default UI direction.]]),
```

Comment:
To do Skim or not to do Skim (and remove it from this help_text), that is the question :)

Reply:
Done